include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -33,14 +33,15 @@ struct __subgroup_radix_sort
     auto
     operator()(sycl::queue __q, _RangeIn&& __src)
     {
+        using _SortKernel_1 = oneapi::dpl::__par_backend_hetero::__internal::__kernel_name_provider<_KernelNameBase, CallNumber>;
```

Comment:
Maybe add a comment that positive `CallNumber` is for local memory and negative `CallNumber` is for global memory.

Reply:
is not actual anymore, the code was changed